### DISCUSSION TITLE
INDEX ERROR

### DISCUSSION THREAD

**Discussion by Xiaodu1111** [microsoft/graphrag#461] [Q&A]
Has anyone else encountered this error? How do I fix it?
![1](https://github.com/microsoft/graphrag/assets/127846766/0abbe911-91f0-4e4a-abc4-ef690b63ec73)
![2](https://github.com/microsoft/graphrag/assets/127846766/7ab02040-6ea0-4e45-a29c-f077b90a6f5e)

**Comment by AlonsoGuevara** [comment]
Hi,
This is likely caused due to entity extraction not working. It can be either that the model is replaying with faulty outputs, or that the LLM calls are failing.
Can you please provide your config?
